packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -97,17 +100,14 @@ class AttestationRequestHandler {
     const address = getAccountAddress()
 
     // TODO: Check with the new Accounts.sol
-    if (address.toLowerCase() !== this.attestationRequest.issuer.toLowerCase()) {
+    if (eqAddress(address, issuer)) {
```

Comment:
Should be:
`if (!eqAddress(address, issuer))`

Reply:
Good catch thanks!